### PR TITLE
tidy language on criteria page

### PR DESCRIPTION
**Pull request by clausmullie**

### CODE REVIEW

**Comment by clausmullie** [comment]
Line 60 I changed
4. How realistic vs ambitious are your goals in the next 6 months?
To line 61
5. Would you describe your goals in the next six months as realistic or ambitious?

This probably should have been a suggested change as this changes meaning.

File: activities/codebase-stewardship/criteria-for-codebase-stewardship.md [hunk 1]
```diff
@@ -54,21 +54,21 @@ The current owners of potential codebases are asked to answer these questions to
 
 ### Community potential
 
-1. Is there a 2 to 4 year roadmap?
+1. Is there a two to four year roadmap?
 2. Is there a public backlog in English or one you can make public?
-3. How far through your current set of goals are you?
-4. How realistic vs ambitious are your goals in the next 6 months?
-5. Is there a public bugtracking system in English or one you can make public?
+3. Is there a public bugtracking system in English or one you can make public?
```
Comment (ElenaFdR) [inline activities/codebase-stewardship/criteria-for-codebase-stewardship.md]: The original questions 3 and 4 were about the roadmap and backlog, not the bugtracking system. That's why they were asked in that order.

File: activities/codebase-stewardship/criteria-for-codebase-stewardship.md [hunk 2]
```diff
@@ -54,21 +54,21 @@ The current owners of potential codebases are asked to answer these questions to
...
+4. How far through your current set of goals are you?
+5. Would you describe your goals in the next six months as realistic or ambitious?
```
Comment (ElenaFdR) [inline activities/codebase-stewardship/criteria-for-codebase-stewardship.md]: The original wording was meant to suggest a continuum - people replying in binary isn't as useful to us.

File: activities/codebase-stewardship/criteria-for-codebase-stewardship.md (hunk 1 above)
Comment (clausmullie) [reply activities/codebase-stewardship/criteria-for-codebase-stewardship.md]: aha! Then I would suggest making them either subquestions, or referring back to the backlog directly, to make this clearer

Comment (ElenaFdR) [reply activities/codebase-stewardship/criteria-for-codebase-stewardship.md]: I've put them back into the previous order. I wonder if the source of the confusion is the (deliberate) vagueness about 'your current set of goals'?

**Comment by ElenaFdR** [comment]
@clausmullie I've added questions for Story, Societal Impact and Business Case - take a look?

File: activities/codebase-stewardship/criteria-for-codebase-stewardship.md [hunk 3]
```diff
@@ -54,21 +57,36 @@ The current owners of potential codebases are asked to answer these questions to
 
 ### Community potential
 
-1. Is there a 2 to 4 year roadmap?
+1. Is there a two to four year roadmap?
 2. Is there a public backlog in English or one you can make public?
-3. How far through your current set of goals are you?
-4. How realistic vs ambitious are your goals in the next 6 months?
+3. Are your goals for the next 6 months more ambitious or realistic?
```
Comment (clausmullie) [inline activities/codebase-stewardship/criteria-for-codebase-stewardship.md]:
```suggestion
3. Are your backlog goals for the next 6 months more ambitious or realistic?
```

File: activities/codebase-stewardship/criteria-for-codebase-stewardship.md [hunk 4]
```diff
@@ -54,21 +57,36 @@ The current owners of potential codebases are asked to answer these questions to
...
+4. How far through your current set of goals are you?
```
Comment (clausmullie) [inline activities/codebase-stewardship/criteria-for-codebase-stewardship.md]: 3. Are your backlog goals for the next 6 months more ambitious or realistic?

File: activities/codebase-stewardship/criteria-for-codebase-stewardship.md [hunk 5]
```diff
@@ -19,15 +19,18 @@ These are the evaluation criteria the staff of the Foundation for Public Code us
 
 ## Evaluation questions
 
-The current owners of potential codebases are asked to answer these questions to help us better understand their codebase.
+This is a list of questions to ask the current owners of potential codebases to help us better understand their codebase before the Assess phase.
```
Comment (ElenaFdR) [inline activities/codebase-stewardship/criteria-for-codebase-stewardship.md]:
```suggestion
This is a list of questions for current owners of potential codebases to help us better understand their codebase before the [Assess phase](https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html).
```

**Pull request merged**